src/solvers/dmrg_x_solver.jl

```diff
@@ -0,0 +1,19 @@
+function dmrg_x_solver(
+  init;
+  psi_ref!,
+  PH_ref!,
+  normalize=nothing,
+  region,
+  sweep_regions,
+  sweep_step,
+  half_sweep,
+  step_kwargs...,
```

Comment:
Is this right? I thought we were planning to avoid slurping extra keyword arguments in this way.

Reply:
I haven't yet gotten to update `dmrg_x`